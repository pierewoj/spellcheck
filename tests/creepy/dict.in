insert this
insert month
insert id
insert like
insert to
insert discuss
insert superstitions
insert for
insert those
insert of
insert you
insert who
insert are
insert unfamiliar
insert with
insert the
insert term
insert a
insert superstition
insert is
insert a
insert belief
insert that
insert two
insert things
insert are
insert connected
insert despite
insert a
insert lack
insert of
insert any
insert concrete
insert proof
insert as
insert an
insert example
insert one
insert of
insert the
insert most
insert famous
insert superstitions
insert is
insert that
insert breaking
insert a
insert mirror
insert will
insert curse
insert you
insert with
insert seven
insert years
insert of
insert bad
insert luck
insert theres
insert no
insert true
insert evidence
insert that
insert any
insert actual
insert causation
insert exists
insert here
insert yet
insert this
insert belief
insert is
insert still
insert passed
insert down
insert over
insert generations
insert so
insert tell
insert us
insert do
insert you
insert subscribe
insert to
insert any
insert beliefs
insert that
insert could
insert be
insert called
insert superstitions
insert if
insert so
insert why
insert do
insert you
insert have
insert anecdotes
insert of
insert your
insert superstitions
insert appearing
insert true
insert do
insert you
insert remember
insert when
insert you
insert first
insert learned
insert any
insert of
insert your
insert superstitions
insert if
insert you
insert dont
insert personally
insert believe
insert any
insert superstitions
insert but
insert you
insert have
insert familial
insert or
insert cultural
insert superstitions
insert that
insert youd
insert like
insert to
insert share
insert please
insert do
insert very
insert often
insert these
insert beliefs
insert appear
insert evolve
insert and
insert spread
insert within
insert certain
insert regions
insert or
insert groups
insert and
insert ive
insert always
insert found
insert it
insert fascinating
insert to
insert compare
insert notes
insert if
insert you
insert have
insert any
insert theories
insert or
insert stories
insert about
insert how
insert a
insert specific
insert superstition
insert developed
insert please
insert feel
insert free
insert to
insert share
insert with
insert us
insert the
insert only
insert guidelines
insert here
insert are
insert the
insert general
insert commenting
insert rules
insert be
insert polite
insert and
insert dont
insert use
insert this
insert topic
insert as
insert an
insert excuse
insert to
insert troll
insert or
insert insult
insert each
insert other
insert obligatory
insert soundtrack
insert incoming
save slownik.dict

