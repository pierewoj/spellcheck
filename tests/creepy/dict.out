inserted: this
inserted: month
inserted: id
inserted: like
inserted: to
inserted: discuss
inserted: superstitions
inserted: for
inserted: those
inserted: of
inserted: you
inserted: who
inserted: are
inserted: unfamiliar
inserted: with
inserted: the
inserted: term
inserted: a
inserted: superstition
inserted: is
ignored
inserted: belief
inserted: that
inserted: two
inserted: things
ignored
inserted: connected
inserted: despite
ignored
inserted: lack
ignored
inserted: any
inserted: concrete
inserted: proof
inserted: as
inserted: an
inserted: example
inserted: one
ignored
ignored
inserted: most
inserted: famous
ignored
ignored
ignored
inserted: breaking
ignored
inserted: mirror
inserted: will
inserted: curse
ignored
ignored
inserted: seven
inserted: years
ignored
inserted: bad
inserted: luck
inserted: theres
inserted: no
inserted: true
inserted: evidence
ignored
ignored
inserted: actual
inserted: causation
inserted: exists
inserted: here
inserted: yet
ignored
ignored
ignored
inserted: still
inserted: passed
inserted: down
inserted: over
inserted: generations
inserted: so
inserted: tell
inserted: us
inserted: do
ignored
inserted: subscribe
ignored
ignored
inserted: beliefs
ignored
inserted: could
inserted: be
inserted: called
ignored
inserted: if
ignored
inserted: why
ignored
ignored
inserted: have
inserted: anecdotes
ignored
inserted: your
ignored
inserted: appearing
ignored
ignored
ignored
inserted: remember
inserted: when
ignored
inserted: first
inserted: learned
ignored
ignored
ignored
ignored
ignored
ignored
inserted: dont
inserted: personally
inserted: believe
ignored
ignored
inserted: but
ignored
ignored
inserted: familial
inserted: or
inserted: cultural
ignored
ignored
inserted: youd
ignored
ignored
inserted: share
inserted: please
ignored
inserted: very
inserted: often
inserted: these
ignored
inserted: appear
inserted: evolve
inserted: and
inserted: spread
inserted: within
inserted: certain
inserted: regions
ignored
inserted: groups
ignored
inserted: ive
inserted: always
inserted: found
inserted: it
inserted: fascinating
ignored
inserted: compare
inserted: notes
ignored
ignored
ignored
ignored
inserted: theories
ignored
inserted: stories
inserted: about
inserted: how
ignored
inserted: specific
ignored
inserted: developed
ignored
inserted: feel
inserted: free
ignored
ignored
ignored
ignored
ignored
inserted: only
inserted: guidelines
ignored
ignored
ignored
inserted: general
inserted: commenting
inserted: rules
ignored
inserted: polite
ignored
ignored
inserted: use
ignored
inserted: topic
ignored
ignored
inserted: excuse
ignored
inserted: troll
ignored
inserted: insult
inserted: each
inserted: other
inserted: obligatory
inserted: soundtrack
inserted: incoming
dictionary saved in file slownik.dict
